jwst/assign_wcs/nirspec.py

```diff
@@ -574,7 +620,7 @@ def get_open_msa_slits(msa_file, msa_metadata_id, dither_position,
 
         # subtract 1 because shutter numbers in the MSA reference file are 1-based.
         shutter_id = xcen + (ycen - 1) * 365
-        source_id = slitlets_sid[0]['source_id']
+        #source_id = slitlets_sid[0]['source_id']
```

Comment:
Any reason to comment this out instead of removing the line?

Reply:
Yes, this is handled differently now for source and bkg slitlets and this is done in the two specific code blocks.